src/style.css

```diff
@@ -0,0 +1,92 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300px;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.container {
+  display: flex;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+  left: 26px;
+  bottom: 26px;
+  top: 25px;
+}
+
+.logo-small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  bottom: 16px;
+  top: 15px;
+}
+
+.search_bar {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+}
+
+.search_bar-small {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 14px;
+}
+
+.search_bar:hover,
+.search_bar-small:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search_bar:focus,
+.search_bar-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/190852744-ebeb2fe3-7b31-4b45-b238-afb5d86cc846.png)

Reply:
Thank you! Duly noted. All corrected, please check.